tests/rados/test_cephdf.py

```diff
@@ -286,3 +293,237 @@ def run(ceph_cluster, **kw):
 
         log.info("ceph df MAX AVAIL stats verification completed")
         return 0
+
+    if config.get("cephdf_max_avail_osd_expand"):
+        desc = (
+            "\n#CEPH-83595780"
+            "\nReef: BZ-2296248"
+            "\nSquid: BZ-2296247"
+            "\nThis test is to verify that ceph df MAX AVAIL is displayed correctly for all the pools"
+            " when OSD size is increased"
+            "\nSteps- \n"
+            "1. Creating a pool with default config\n"
+            "2. Log pool stats and verify max_avail \n"
+            "3. Create LVMs on backup nodes \n"
+            "4. Add backup hosts to the cluster and deploy OSDs on them \n"
+            "5. Log pool stats and verify max_avail \n"
+            "6. Expand OSD size on one backup host \n"
+            "7. Log pool stats and verify max_avail \n"
+            "8. Write data to a pool to fill the cluster \n"
+            "9. Log pool stats and verify max_avail \n"
+            "10. Expand OSD size on other backup host \n"
+            "11. Log pool stats and verify max_avail"
+        )
+
+        log.info(desc)
+        df_config = config.get("cephdf_max_avail_osd_expand")
+        pool_name = "test-osd-expand"
+        lvm_list = {"node12": [], "node13": []}
+
+        def bytes_to_gb(val):
+            return round(val / (1 << 30), 1)
+
+        try:
+            # pass without execution for Squid
```

Comment:
please create a backlog task to track the fix for other versions as well. If no backport bug exists for quincy and if the issue exists, please raise a backport bug.